sveltekit/src/routes/home.svelte

```diff
@@ -1,31 +0,0 @@
-<script context="module">
```

Comment:
Removing this file will break the CLI, because it's referenced here as the file to load and replace for the bridge: https://github.com/storyblok/storyblok-create-demo/blob/main/src/frameworks.js#L68

Reply:
exactly @lisilinhart . We have to change
```
bridge: 'src/routes/home.svelte',
```
to
```
bridge: 'src/routes/index.svelte',
```
do you want that I push a PR on the create-demo repo?
I was aligned also with @josefineschaefer  to be sure that all svelte documentation refers to index.svelte (the default for svelte apps) and not home.svelte)

Reply:
No, it needs a story on the `/home` path, not the `/` path, because in new spaces, we will not have the `realPath` set and we are directly redirecting into that story and it should work without setting the `realPath` first. So either it was a `home.svelte` page or some kind of automatic slug handling

Reply:
You can try the CLI to check the behaviour of how it works